### PR TITLE
updates readme with Node Girls-specific information

### PR DESCRIPTION
**Pull request by msmichellegar**
I've updated the Readme description from the fork with a more Node Girls-specific introduction to the Code of Conduct.

### CODE REVIEW

**Comment by msmichellegar** [comment]
Anyone want to review? @nikhilaravi @nofootnotes @minaorangina @nikhilaravi @iteles

File: README.md [hunk 1]
```diff
@@ -1,14 +1,10 @@
-confcodeofconduct.com
-=====================
+# Code of confcodeofconduct
```
Comment (iteles) [inline README.md]: I think we could just have 'Code of Conduct' here :blush:

File: README.md [hunk 2]
```diff
@@ -1,14 +1,10 @@
...
-------------
+This is the Node Girls Code of Conduct, forked from the original [Conf Code of Conduct](http://confcodeofconduct.com/). We'll be enforcing this Code of Conduct at all Node Girls events, as well as in our online community.
 
-If you would like to contribute a translation to this project, please include a translated file that includes the language short code in the filename as this will be mapped to from the url. For example, fr.confcodeofconduct.com will load the `index-fr.html` file.
+If you would like to let us know about any violations of the Code of Conduct, please get in touch with us via nodegirlslondon@gmail.com, or speak to one of the chapter organisers.
 
-Fork this project, send a pull request and the translation will go live (though the translation may be validated before publishing).
-
-Please check the [issues](https://github.com/leftlogic/confcodeofconduct.com/issues) to see if anyone else is currently working on a translation.
+## Translations
```
Comment (iteles) [inline README.md]: Could we move English to be at the top of the list?

Comment (msmichellegar) [inline README.md]: Good point. Actually, I was thinking about putting the entire English version of the code of conduct in the README. What do you think?

Comment (iteles) [inline README.md]: That's a good idea although that also means the burden is on you to ensure it's up to date ;)

File: README.md (hunk 1 above)
Comment (NataliaLKB) [inline README.md]: Yes I think that just `Code of Conduct` here is good.

**Comment by NataliaLKB** [comment]
Other then the comments, all looks good!

**Comment by msmichellegar** [comment]
Have updated the Code of Conduct! Want to review? @NataliaLKB @iteles

**Comment by iteles** [comment]
Looks great! Thanks @msmichellegar :ok_hand:

**Pull request merged**